src/main/kotlin/org/sert2521/bunnybots2019/beddumper/BedDumperCommands.kt

```diff
@@ -0,0 +1,37 @@
+package org.sert2521.bunnybots2019.beddumper
+
+import kotlinx.coroutines.launch
+import org.sert2521.sertain.coroutines.periodic
+import org.sert2521.sertain.events.onTick
+import org.sert2521.sertain.subsystems.doTask
+import org.sert2521.sertain.subsystems.use
+
+suspend fun beddumper() = doTask {
+    val beddumper = use<Beddumper>()
+    action {
+        try {
+            periodic(20) {
+                beddumper.spinBedDumper()
+                println("Spinning Bed Dumper")
+            }
+        } finally {
+            println("Bed Dumper should stop")
+            beddumper.stopSpin()
+        }
+    }
+}
+
+//suspend fun reverseIntake() = doTask {
```

Comment:
More comments.

However, That is a great learning strategy to copy similar code but make it do something slightly different!